resteasy/src/test/java/com/cerner/beadledom/resteasy/TestJaxRsProvider.java

```diff
@@ -0,0 +1,16 @@
+package com.cerner.beadledom.resteasy;
+
+import javax.ws.rs.ext.Provider;
+
+/**
+ * JAX-RS provider for testing.
```

Comment:
Why do you add your `@author` tag sometimes and not others?

Reply:
https://github.com/cerner/beadledom/pull/143/commits/75d364f068be5b48f12a864aa2a8bd938fb929b8